src/test/api/Authentication.test.js

```diff
@@ -0,0 +1,32 @@
+import { expect } from 'chai';
+import { Authentication } from '../../api/Authentication';
+
+describe('Authentication Service Api' , () => {
+  it('returns successful on valid email and password', () => {
+    expect(Authentication.validityChecker('abc@gmail.com', 'abc')).to.equal('Successful');
```

Comment:
***Code Quality: Awesome/Nitpick***
Excellent tests here. are we not suppose to test that localforage.setItem is called by spying on the setItem property?